server/src/QRServer/api/api.py

```diff
@@ -60,19 +65,96 @@ def _add_routes(self):
             web.get('/api/v1/game/stats', self._v1_game_stats),
             web.get('/api/v1/health', self._v1_health),
             web.get('/api/v1/lobby/stats', self._v1_lobby_stats),
+            web.get('/api/v1/tournaments/{id}', self._v1_tournaments),
+            web.get('/api/v1/tournaments/{id}/users', self._v1_tournament_users),
+            web.get('/api/v1/tournaments/{id}/duels', self._v1_tournament_duels),
+            web.get('/api/v1/tournaments/{id}/matches', self._v1_tournament_matches),
         ])
 
-    async def _v1_game_stats(self, _request: web.Request):
+    async def _v1_game_stats(self, _request: web.Request) -> web.Response:
         player_count = self.game_server.get_player_count()
         return web.json_response({
             'player_count': player_count,
         })
 
-    async def _v1_health(self, _request: web.Request):
+    async def _v1_health(self, _request: web.Request) -> web.Response:
         return web.json_response({})
 
     async def _v1_lobby_stats(self, _request: web.Request):
         player_count = self.lobby_server.get_player_count()
         return web.json_response({
             'player_count': player_count,
         })
+
+    async def _v1_tournament_users(self, request) -> web.Response:
+        tournament_id = request.match_info['id']
+        users: list[DbUser] = await self.db_conn.list_tournament_users(tournament_id)
+
+        if not users:
+            return web.json_response(data={}, status=200)
+
+        users_view = {}
+
+        for user in users:
+            user_view = {
+                'id': user.user_id,
+                'username': user.username,
+            }
+            users_view[user.user_id] = user_view
```

Comment:
1. It should be a list, not a map
2. It's best not to return a list directly, as it's not extensible

Reply:
ok